Cargo.toml

```diff
@@ -63,15 +62,14 @@ rpc = ["bitcoincore-rpc"]
 test-blockchains = ["bitcoincore-rpc", "electrum-client"]
 test-electrum = ["electrum"]
 test-rpc = ["rpc"]
-test-esplora = ["esplora"]
+test-esplora = ["esplora", "electrsd/legacy"]
 test-md-docs = ["electrum"]
 
 [dev-dependencies]
 lazy_static = "1.4"
 env_logger = "0.7"
 clap = "2.33"
-serial_test = "0.4"
-bitcoind = "0.10.0"
+electrsd = { version="0.4.0", features = ["trigger"] }
```

Comment:
nit: use `0.4` instead so that we can push patch-level updates if we need to

Reply:
48dcc45b9b7f7551411108e70ed9121e90bacd44